java/Makefile

```diff
@@ -26,6 +26,9 @@ classes.jar: $(SRCS) $(JPP_DESTS) $(EXTRA)
 	$(foreach dir,$(SRC_DIRS),cp -a $(dir)/. build-src/;)
 	find ./build-src -name *.java > build-srcs.txt
 	javac -cp build-src -g:none -source 1.3 -target 1.3 -bootclasspath "" -extdirs "" -d ./build @build-srcs.txt > /dev/null
+	java -jar ../build_tools/soot-trunk.jar -j2me -process-dir build -no-output-source-file-attribute -no-output-inner-classes-attribute -force-overwrite -include-all 2>&1 > /dev/null
+	cp -a sootOutput/. build/
+	rm -rf sootOutput
```

Comment:
> Done in 46f5e2d, the files in these directories are already ignored (because in .gitignore we have a rule for java/build-src/ and a rule for *.class)

Looks good, thanks!